### ISSUE TITLE
Version 2.6.0 introduces a breaking change

### ISSUE THREAD

**Issue by JamieDixon** [tomchentw/react-toastr#49]
Since the 2.6.0 release, React 15 is required.

This broke our codebase because 2.6.0 introduces a breaking change and our match for "^2.4.0" allowed for 2.6.0 to be installed without warning.

Since 2.6.0 introduces this breaking change I suggest the version be bumped to the next major version, 3.0 in line with semver.

What do you think? Thanks for react-toastr, we really appreciate all your work on this library!

**Comment by jerairrest** [comment]
I agree!

**Comment by tomchentw** [comment]
Apologies for that. I didn't review carefully when merging PR #48. How about change it to:

```js
    "react": "^0.14.0 || ^15.0.0",
    "react-dom": "^0.14.0 || ^15.0.0"
```

**Comment by tomchentw** [comment]
* fixed in 394ef07
* marked `v2.6.0` as `deprecated`
* published [v2.6.1](https://github.com/tomchentw/react-toastr/releases/tag/v2.6.1)

**Comment by JamieDixon** [comment]
Thanks Tom. Much appreciated. Be well.
